integration-tests/runtime-tests/src/types/interfaces/augment-api-consts.ts

```diff
@@ -198,8 +205,49 @@ declare module '@polkadot/api-base/types/consts' {
        **/
       [key: string]: Codec;
     };
+    lending: {
```

Comment:
Does this PR combines changes related to lending pallets too? if not please remove these changes.

Reply:
Just ignore this file. It's part of the typescript types for our chain and gets automatically generated.